### ISSUE TITLE
iCloud sync

### ISSUE THREAD

**Issue by flcouselo** [a-busman/Backlogger#49]
It would be great to have actual iCloud sync, without any user interaction each time you add/edit a game in the app. The export/import feature is nice to have but a bit outdated and you always run the risk of forgetting to make a backup and losing the latest data.

**Comment by a-busman** [comment]
I agree that this is less than ideal. I'll have to figure out a way to get Realm to just go to iCloud easily.

**Comment by a-busman** [comment]
The Realm way of doing this is to have your own object server, something I am unwilling to pay for. I can either change my db to use CoreData and then put that into iCloud, or translate Realm to CloudKit records.

**Comment by a-busman** [comment]
Moving this to v2.0. Will implement with CoreData and CloudKit

**Comment by a-busman** [comment]
Cloudkit not being implemented for 2.0. Will put in backlog
